### ISSUE TITLE
Tree view doesn't move to new window

### ISSUE THREAD

**Issue by rp8085** [atom/tree-view#126]
If I have 2 Atom windows open and drag a tab from one window to another, the tree view  stays behind, even when there are no other files open in the first window. So now I have the file and its corresponding tree view open in different windows. I tried toggling the tree view off/on but that doesn't help.

**Comment by bronson** [comment]
Yes, it looks like tree-view is disabled/broken for any new windows that are opened.  This seems really weird to me -- why should the first window be special?  And, if I close the first window, then I don't get to use the tree-view until I restart the editor!

To reproduce:
- Open a new window
- Try to show the tree view

No tree view for you!

It seems like this is the same bug...  if not, I'm happy to open a new one.
